src/components/RestaurantCard.js

```diff
@@ -0,0 +1,29 @@
+import { Restaurants } from './restaurants';
+import styled from 'styled-components';
+
+export default function RestaurantCard() {
+  const randomRestaurant =
+    Restaurants[Math.floor(Math.random() * Restaurants.length)];
```

Comment:
Variables starting with an uppercase letter are reserved for Classes (which we didn't cover in the course) and React Components. Here you should use 'restaurants' instead.